manager/integration/tests/test_ha.py

```diff
@@ -2141,6 +2141,46 @@ def test_auto_remount_with_subpath(client, core_api, storage_class, sts_name, st
     assert expect_md5sum == md5sum
 
 
+@pytest.mark.skip(reason="WAIT_FOR_UPSTREAM_FIX") # NOQA
+@pytest.mark.skipif( "k3s" not in k8s_api_client.status.node_info.kubelet_version, reason='This test is only compaitable with k3s')  # NOQA
```

Comment:
Is `k8s_api_client.status.node_info.kubelet_version` available before our test env is set up? If not, the pdoc check will fail. 
I think, it's better if we include the check in the test.

Reply:
Okay, I will check again, and see if we want a variable that stores the cluster version before the test case.

Reply:
Thanks Chris for the hint. In latest commit, I am using subprocess to call kubectl and obtain the node version.